### DISCUSSION TITLE
花火

### DISCUSSION THREAD

**Discussion by Klein124284420** [share121/inter-knot#4511] [崩坏：星穹铁道]
![u_2547698720_244668838 fm_30 app_106 f_JPEG](https://github.com/user-attachments/assets/35ebf035-9ce6-40fe-b18b-e9cdfb4aeb95)
好看

**Comment by XMSTRIKEGP** [comment]
豪堪！
![Uploading 6d710c12b07eca80fecd05d2d72397dda0448399.jpg…]()
